packages/purpose-template-process/src/model/domain/errors.ts

```diff
@@ -36,8 +36,7 @@ export const errorCodes = {
   annotationDocumentLimitExceeded: "0019",
   conflictDuplicatedDocument: "0020",
   hyperlinkDetectionError: "0021",
-  purposeTemplateNotInValidState: "0022",
-  invalidAssociatedEServiceForPublicationError: "0023",
+  invalidAssociatedEServiceForPublicationError: "0022",
```

Comment:
Here it could be OK to delete the unused error code, but I would avoid changing the mapping (here `invalidAssociatedEServiceForPublicationError` is changed from `0022` to `0023`). That  would be risky if someone relies on the current error code (unlikely for `purpose-template-process`, which is new, but could happen in other similar cases I guess). Leaving a gap (since no error would be mapped to `0022`) isn't pretty but shouldn't break things.

What do you think about this?